crates/core/tedge_api/src/lib.rs

```diff
@@ -16,6 +16,8 @@ pub mod utils;
 
 pub use download::*;
 pub use error::*;
+pub use health::health_status_down_message;
+pub use health::health_status_up_message;
```

Comment:
It really feels like these 2 functions better belong in the `tedge_health_ext` crate itself as every plugin wanting the health-check functionality is expected to depend on this crate. But, since all the older plugins are also relying on the same, we can consider moving these in future.